pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -1097,47 +1085,7 @@ type GettableApiReceiver struct {
 
 func (r *GettableApiReceiver) UnmarshalJSON(b []byte) error {
 	type plain GettableApiReceiver
-	if err := json.Unmarshal(b, (*plain)(r)); err != nil {
-		return err
-	}
-
-	hasGrafanaReceivers := len(r.GrafanaManagedReceivers) > 0
```

Comment:
this removes an obsolete check on a read-only model. So, it basically validates what's in the source of data